rs/protobuf/def/state/queues/v1/queues.proto

```diff
@@ -54,8 +54,11 @@ message StreamEntry {
 }
 
 message RequestMetadata {
-  optional uint64 call_tree_depth = 1;
-  optional uint64 call_tree_start_time_nanos = 2;
+  // TODO(MR-641): Remove deprecated fields.
+  optional uint64 deprecated_opt_call_tree_depth = 1;
+  optional uint64 deprecated_opt_call_tree_start_time_nanos = 2;
+  uint64 call_tree_depth = 4;
+  uint64 call_tree_start_time_nanos = 5;
```

Comment:
I don't think this is necessary. The difference between `optional uint64` and `uint64` is that the former explicitly encodes a value of zero, whereas the latter just skips encoding it (producing zero bytes).

So from the point of view of the encoded data, it should be entirely safe to just drop the `optional` qualifier.

Reply:
I get one failing test that may be due to me trying it with just removing `optional` here: //pre-commit:buf-breaking
It passes when I run it locally though. Not sure what to make of it.

Reply:
The exception has fixed the tests. I take it this means it should be ok to just remove the 'optional' from the protobuf definitions.